### PR TITLE
Feature/tg 100 create meeting

### PR DESCRIPTION
**Pull request by carlosBarriosCarvajal**
# Summary

Create service for creating a single meeting for a classroom

# Details

* Added migration for seeding MeetingAttendanceStatus
* Added endpoint for meeting creation
* Create service for single meeting creation
* Added tests

# Evidence
Captura de pantalla 2024-02-22 181901

Captura de pantalla 2024-02-22 181159

Captura de pantalla 2024-02-22 182343
Captura de pantalla 2024-02-22 182432

### CODE REVIEW

File: KlockanAPI/tests/KlockanAPI.Application.Tests/Services/MeetingServiceTests.cs [hunk 1]
```diff
@@ -80,4 +81,47 @@ public async Task GetAllMeetingsAsync_ShouldReturnMeetingDto()
...
+        _meetingRespository.CreateSingleMeeting(Arg.Any<Meeting>()).Returns(createdMeeting);
+
+        // Act
+        var result = await meetingService.CreateSingleMeeting(createMeetingDto);
+
+        // Assert
+        result.Should().NotBeNull();
+        result.Should().BeEquivalentTo(expectedMeeting);
+    }
 }
```
Comment (AJN-jalaU) [inline KlockanAPI/tests/KlockanAPI.Application.Tests/Services/MeetingServiceTests.cs]: please add empty line at the end of the file

File: KlockanAPI/src/KlockanAPI.Presentation/Controllers/MeetingsController.cs [hunk 2]
```diff
@@ -26,4 +26,33 @@ public async Task<ActionResult<IEnumerable<MeetingDto>>> GetAllMeetings()
...
+        }
+        catch (Exception ex)
+        {
+            return StatusCode(500,$"Internal server error: {ex.Message}");
+        }
+
+
+
+    }
 }
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Presentation/Controllers/MeetingsController.cs]: please add empty line at the end of the file

File: KlockanAPI/src/KlockanAPI.Application/Services/Interfaces/IMeetingService.cs [hunk 3]
```diff
@@ -4,4 +4,5 @@ namespace KlockanAPI.Application.Services.Interfaces;
 public interface IMeetingService
 {
     Task<IEnumerable<MeetingDto>> GetAllMeetingsAsync();
+    Task<MeetingDto> CreateSingleMeeting(CreateMeetingDto meeting);
 }
```
Comment (AJN-jalaU) [inline KlockanAPI/src/KlockanAPI.Application/Services/Interfaces/IMeetingService.cs]: please add empty line at the end of the file

File: KlockanAPI/tests/KlockanAPI.Presentation.Tests/Controllers/MeetingsControllerTests.cs [hunk 4]
```diff
@@ -75,4 +77,56 @@ public async Task GetAll_ShouldReturnOk()
...
+        };
+
+        controller.ModelState.AddModelError("PropertyName", "Error message");
+
+        // Act
+        var result = await controller.CreateMeeting(createMeetingDto);
+
+        // Assert
+        Assert.IsType<BadRequestObjectResult>(result.Result);
+    }
```
Comment (AJN-jalaU) [inline KlockanAPI/tests/KlockanAPI.Presentation.Tests/Controllers/MeetingsControllerTests.cs:131]: please add empty line at the end of the file

**Review by AJN-jalaU — COMMENTED** [review]
Could you add the image of the line coverage of the files/ methods you create?

**Review by AJN-jalaU — APPROVED** [review]

**Review by pierrecf — APPROVED** [review]
Approved :D

**Pull request merged**